package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "ourjseditor",
+  "version": "1.0.0",
+  "description": "This repository is for storing the server-side code which powers [OurJSEditor.com](https://ourjseditor.com).",
+  "scripts": {
+    "build": "webpack",
+    "errorDetails": "webpack --display-error-details",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/OurJSEditor/OurJSEditor.git"
+  },
+  "author": "",
+  "bugs": {
+    "url": "https://github.com/OurJSEditor/OurJSEditor/issues"
+  },
+  "homepage": "https://github.com/OurJSEditor/OurJSEditor#readme",
+  "dependencies": {
+    "promise-polyfill": "^8.1.0",
+    "react": "^16.8.1",
+    "react-dom": "^16.8.1",
+    "whatwg-fetch": "^3.0.0"
```

In order to get this to compile, I had to install `babel-cli` and `babel-preset-es2015`. Now, Babel recommends using `babel-preset-env` over `babel-preset-es2015`, and `babel-preset-env` is linked in dev dependencies. I think [this](https://github.com/OurJSEditor/OurJSEditor/pull/90/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR27) code from webpack.config could also be changed to use `babel-preset-env` instead of `babel-preset-es2015`, but I'm not sure. 

```suggestion
    "whatwg-fetch": "^3.0.0",
    "babel-cli": "^6.26.0",
    "babel-preset-es2015": "^6.24.1"
```